src/PlaneFitting.hpp

```diff
@@ -89,25 +89,25 @@ class PlaneFitting
 
     class Result
     {
-	Matrix3 A;
```

Comment:
Ah ... Yes... This is a class. Private by default and stuff. I'm in need of more sleep.